src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -706,6 +722,14 @@ uct_ud_verbs_iface_post_recv_always(uct_ud_verbs_iface_t *iface, int max)
         return;
     }
 
+    /* Set the first byte in the receive buffer grh to a known value not equal to
+     * 0x6?. This should aid in the detection of IPv6 vs IPv4 because the first
+     * byte is undefined in the later and 0x6? in the former. It is unlikely
+     * this byte is touched with IPv4. */
+    for (i = 0 ; i < count ; ++i) {
```

Comment:
our coding style:
```suggestion
    for (i = 0; i < count; ++i) {
```

Reply:
Sure. I am running that now. Did not realize there was a clang format file. Will update this shortly.

Reply:
Done.

Reply:
Or not. One second.